home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -36,6 +36,12 @@ tags:
     description: Cooperation poll management operations
   - name: poll
     description: Poll management operations  independently from cooperation
+  - name: cooperation contact
```

Comment:
Do we really need these new tags?

Reply:
> Do we really need these new tags?

This is already existing tag that was previously forgotten to be added to the list of common tags. I just added it to the "tag" section and added a description that was not there before.